API.yaml

```diff
@@ -349,6 +441,76 @@ definitions:
         minimum: 0
         maximum: 4096
         example: 4
+  OperationalInfo:
+    type: "object"
+    required:
+    - "province"
+    - "exposure_permission"
+    - "bluetooth_active"
+    - "notification_permission"
+    - "exposure_notification"
+    - "last_risky_exposure_on"
+    properties:
+      province:
+        type: "string"
+        pattern: "^[A-Z]{2}$"
+        example: "MB"
+      exposure_permission:
+        type: "integer"
+        format: "int32"
+        minimum: 0
+        maximum: 1
+        example: 1
+      bluetooth_active:
+        type: "integer"
+        format: "int32"
+        minimum: 0
+        maximum: 1
+        example: 1
+      notification_permission:
+        type: "integer"
+        format: "int32"
+        minimum: 0
+        maximum: 1
+        example: 1
+      exposure_notification:
+        type: "integer"
+        format: "int32"
+        minimum: 0
+        maximum: 1
+        example: 1
+      last_risky_exposure_on:
+        type: "string"
+        format: "date"
+  GoogleOperationalInfo:
+    allOf:
+    - $ref: '#/definitions/OperationalInfo'
+    - type: "object"
+      required:
+      - "salt"
+      - "signed_attestation"
```

Comment:
I'm wondering if wouldn't be better to use this parameter as authorization token like analytics_bearer, but maybe signed attestation is too long for the header.

Reply:
Yes, they are quite long, and unfortunately there is no official documentation on the size of signed attestations. We observed they are currently ~5K characters, but it can be changed by Google in the future. The same applies to the Apple DeviceCheck tokens, which are currently 3K characters, but there is no official indication of their maximum size. We are currently using a configurable upper bound limit of 10K.

Reply:
In this case the body is ok

Reply:
In this case is ok in the body